tmt/utils.py

```diff
@@ -4421,6 +4421,35 @@ def normalize_shell_script_list(
     raise NormalizationError(key_address, value, 'a string or a list of strings')
 
 
+def normalize_shell_script(
+        key_address: str,
+        value: Union[None, str],
+        logger: tmt.log.Logger) -> ShellScript:
+    """
+    Normalize a single string input value.
+
+    This is a fairly common input format present mostly in fmf nodes where
+    tmt, to make things easier for humans, allows this:
+
+    .. code-block:: yaml
+
+       foo: bar
+
+    Internally, we should stick to one type only, and make sure whatever we get
+    on the input, a string would be the output.
+
+    :param value: input value from key source.
+    """
+
+    if value is None:
+        return ShellScript('')
```

Comment:
Maybe we should return `Optional[ShellScript]`. It's what the original lambda did, and `ShellScript('')` seems to be opening the door to tricky issues to investigate (`ShellScript('ls -al /')` + `ShellScript('')` would lead to `ls -al /; ;` which would Bash die on.

Reply:
Right here we should return `None`?

Reply:
Updated. Don't know we should add one simple case for this, if a test is required, please let me know, I'll add it tomorrow.